docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,93 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction convers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. 
+If you already have a downloaded MLCommons repository you can go right away to **Step 5**.
+
+**Step 1**. Clone the MLCommons repository for inference:
```

Comment:
Is it required to clone the whole repository to run 1 model? Is it possible to download only model?

Reply:
Fortunately found a way to pull only the model we need. Updated description accordingly